std/zig/ast.zig

```diff
@@ -45,6 +45,7 @@ pub const Tree = struct {
         line_end: usize,
     };
 
+    /// Return the Location of the token relative to a line that starts at self.source[start_index].
```

Comment:
This is not quite right. It returns the location of the token relative to the offset specified by `start_index`.